AnnoyancesFilter/sections/widgets.txt

```diff
@@ -69,6 +69,8 @@ blog.livedoor.jp##.RSS04 + div[style^="width:720px;height:160px"]
 anacap.doorblog.jp##.blogroll-list-wrap
 anacap.doorblog.jp##.column-inner-2 > center > table[border="1"]:first-child
 anacap.doorblog.jp##div[class^="anacap-roll"]
+healthylives.tw###fbVideo
```

Comment:
You can remove element using extended CSS(`#$?# *** { remove: true; }`) to avoid loading of video(+3.5 MB).